### PR TITLE
TiKV control: Replace `--db `with `--data-dir`

### PR DESCRIPTION
**Pull request by en-jin19**
What is changed, added or deleted? (Required)
- Replaced `--db `with `--data-dir` in commands
- Add a missed part 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7110
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-control.md [hunk 1]
```diff
@@ -226,6 +226,29 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
...
 
+### 扫描 raw key
+
+使用 `raw-scan` 命令可以直接从 RocksDB 进行扫描。
+
+> **注意：**
+>
+> 当要扫描数据 key 时，需要在 key 前添加 `'z'` 前缀。
+
+`raw-scan` 命令的 `--from` 和 `--to` 参数可指定扫描范围（默认为无范围），`--limit` 参数可限制能够打印出的 key 的数量（默认为 `30`），`--cf` 参数可指定进行扫描的 CF（可选  `default`，`write` ，`lock`）。
```
Comment (5kbpers) [inline tikv-control.md:237]:
```suggestion
`raw-scan` 命令的 `--from` 和 `--to` 参数可指定扫描范围（默认不限范围），`--limit` 参数可限制能够打印出的 key 的数量（默认为 `30`），`--cf` 参数可指定进行扫描的 CF（可选  `default`，`write` ，`lock`）。
```

**Review by 5kbpers — COMMENTED** [review]
rest LGTM

**Review by 5kbpers — APPROVED** [review]

File: tikv-control.md [hunk 2]
```diff
@@ -226,6 +226,29 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
 >
 > 该命令中，key 同样需要是 escaped 形式的 raw key。
 
+### 扫描 raw key
+
+使用 `raw-scan` 命令可以直接从 RocksDB 进行扫描。
```
Comment (TomShawn) [inline tikv-control.md:231]:
```suggestion
使用 `raw-scan` 命令可以直接在 RocksDB 中扫描 raw key。
```

File: tikv-control.md [hunk 3]
```diff
@@ -226,6 +226,29 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
...
+
+> **注意：**
+>
+> 当要扫描数据 key 时，需要在 key 前添加 `'z'` 前缀。
```
Comment (TomShawn) [inline tikv-control.md:235]:
```suggestion
> 如果要扫描数据 key，需要在 key 前添加 `'z'` 前缀。
```

File: tikv-control.md [hunk 4]
```diff
@@ -226,6 +226,29 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
...
+
+`raw-scan` 命令的 `--from` 和 `--to` 参数可指定扫描范围（默认不限范围），`--limit` 参数可以限制能够打印出的 key 的数量（默认为 `30`），`--cf` 参数可以指定进行扫描的 CF（可选值为 `default`，`write`，`lock`）。
```
Comment (TomShawn) [inline tikv-control.md:237]:
```suggestion
- 要指定扫描范围，可在 `raw-scan` 命令中使用 `--from` 和 `--to` 参数（默认不限范围）
- 要限制能够打印出的 key 的数量（默认为 `30`），可在命令中使用 `--limit` 参数
- 要指定扫描的 CF，可在命令中使用 `--cf` 参数（可选值为 `default`，`write`，`lock`）
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**